dive_core/command_hierarchy.cpp

```diff
@@ -2547,7 +2547,11 @@ void CommandHierarchyCreator::AddConstantsToPacketNode(const IMemoryManager &mem
             {
                 T        value;
                 uint64_t addr = ext_src_addr + ((i + j) * sizeof(T));
-                DIVE_VERIFY(mem_manager.CopyMemory(&value, submit_index, addr, sizeof(T)));
+
+                // For some reason, some captures refer to memory not backed by memory blocks
+                // Let's not treat it as an error, since cffdump handles this gracefully as well
+                if (!mem_manager.CopyMemory(&value, submit_index, addr, sizeof(T)))
```

Comment:
should we use `DIVE_LOG` (or maybe other func) to print warning messages in this case?

Reply:
Added log as suggested